source/renderer/app/components/profile/bug-report/BugReportDialog.js

```diff
@@ -236,6 +236,11 @@ export default class BugReportDialog extends Component<Props, State> {
     this.setState({ showLogs: value });
   };
 
+  onClose = () => {
+    const { error, onCancel } = this.props;
+    if (!this.state.isSubmitting || error) onCancel();
```

Comment:
@daniloprates closing should be possible in case of an error - we should prevent it only in case of submitting state!